source/declarations.tex

```diff
@@ -3084,7 +3084,7 @@
 \end{note}
 
 \pnum
-An object of type ``array of \tcode{N} \tcode{U}'' contains
+An object of type ``array of \tcode{N} \tcode{U}'' consists of
 a contiguously allocated non-empty set
```

Comment:
Is «contiguously allocated» normative? If not, could it be fixed editorially in this PR or it should be a separate PR?

Reply:
What's wrong with "contiguously allocated"?

Reply:
I've thought we agreed in core reflector that it is not normative, [like the wording about how member subobjects are allocated](https://github.com/cplusplus/draft/commit/bb641cba40211374d8b2dd3472f42274de662761).

Reply:
I don't remember. Do you have a pointer?

Reply:
https://lists.isocpp.org/core/2020/01/8112.php

Reply:
The participation on the core reflector was low for this question. I'd like to restrict this pull request to the obvious fix "contains" -> "consists of".